<!DOCTYPE html>
<!--
    Plain-Academic by Vasilios Mavroudis
    Released under the Simplified BSD License/FreeBSD (2-clause) License.
    https://github.com/mavroudisv/plain-academic
-->
<html lang="en">
<head>
	<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-253454214-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-253454214-1');
</script>

  <title>Jun Bai</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.css" />
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.js"></script>
  <link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css' />
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">	
</head>
<body>


<!-- Navigation -->
	<nav class="navbar navbar-inverse navbar-static-top" role="navigation">
	  <div class="container">
		<div class="navbar-header">
		  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
		  </button>
		</div>

		<!-- Collect the nav links, forms, and other content for toggling -->
		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		  <ul class="nav navbar-nav">
				  <li><a href="index.html">Home</a></li>
			  	  <li><a href="#teams">Teams</a></li>
				  <li><a href="#research">Research</a></li>
				  <li><a href="#publications">Publications</a></li> 
				  <li><a href="#committees">Service</a></li> 
			  	   <li><a href="#teaching">Teaching</a></li> 
		  </ul>
		</div>
	  </div>
	</nav>
  
  <!-- Page Content -->
<div class="container">
        <div class="row">
		<div class="col-md-8">
			<div style="margin-top:3%; text-align:justify;">                
				<p>We’re a research group at University Cincinnati Computer Science Department, 
				and we're using artificial intelligence to push the boundaries of healthcare. 
				Our work dives into AI-powered immune checkpoint inhibitor discovery, analyzing genomic data, 
				and figuring out better ways to spot potential treatment targets. We’re also working on improving how we 
				detect cancer through smarter medical image analysis. In the big picture, we're aiming to help make 
				precision medicine more personal, more effective, and more accessible for everyone.</p>						
			</div>
		</div>
            <!-- Entries Column -->
	        <div class="col-md-2"">	                
	                <!-- Main Image -->
	                <img class="img-responsive" src="profile.png" style="margin-top:3%; alt="" /><br />
	                <div style="font-family: 'Oswald', sans-serif; font-size: 20px;"><b>Dr. Jun Bai</b></div><br />
			<p>Assistant Professor<br />    
	                <p>baiju at ucmail.uc.edu<br />
	
			<P>
				<a href="https://www.linkedin.com/in/jun-b-559b1a95/" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin-square" style="font-size:30;color:blackpx"></i></a>&nbsp &nbsp &nbsp &nbsp
	             		<a href="https://scholar.google.com/citations?user=pWw5QUwAAAAJ&hl=en" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar" style="font-size:30px;color:black"></i></a>	&nbsp &nbsp &nbsp &nbsp   
				<a href="https://github.com/LabJunBMI" title="Github" target="_blank" rel="noopener noreferrer"><i class="fa fa-github" style="font-size:30px;color:black""></i></a>
	        </div>                
	</div>             


	<h2 id="teams">Teams</h2>
	 <div class="row">
		<div class="col-md-2">    
	              <img class="img-responsive" src="poyu.png" style="margin-top:3%; alt="" /><br />
		      <div style="font-family: 'Oswald', sans-serif; font-size: 12px;">Po-Yu Liang</div><br />							 
		</div>
	 
		<div class="col-md-8">
			<p>Po-Yu’s project focuses on developing innovative computational methods to design and 
				generate the structures and sequences of immune checkpoint inhibitors—paving the 
				way for more effective cancer immunotherapies.</p>
		</div>
	</div>
	<div class="row">
		<div class="col-md-2">    
	              <img class="img-responsive" src="wei.png" style="margin-top:3%; alt="" /><br />
		      <div style="font-family: 'Oswald', sans-serif; font-size: 12px;">Wei Wang</div><br />							 
		</div>
	 
		<div class="col-md-8">
			<p>Wei’s project is developing new computaional methods to pinpoint promising therapeutic targets by combining 
				insights from multiple types of genomic data—opening the door to more precise and personalized treatments.</p>
		</div>
	</div>
	
            
		
            <!-- Publications -->
        <div class="col-md-8">    
                <h2 id="publications">Selected publications</h2>                
  		<script src="https://cdn.jsdelivr.net/npm/bibtex-js@2.0.0/bibtex.min.js"></script>
		  <style>
		    body { font-family: Arial, sans-serif; margin: 40px; }
		    h2 { color: #333; }
		    .bibtex_template { margin-bottom: 20px; }
		  </style>
		    <div class="bibtex_template">
		      <p>
		        <span class="if author"><strong><span class="author"></span></strong>.</span><br>
		        <span class="if title"><span class="title"></span>.</span><br>
		        <span class="if journal"><em><span class="journal"></span></em>,</span>
		        <span class="if year"><span class="year"></span>.</span>
		      </p>
		    </div>
		  <script type="text/bibtex">
		    @article{wang2021single,
		        title={Single-cell classification using graph convolutional networks},
		        author={Wang, Tianyu and Bai, Jun and Nabavi, Sheida},
		        journal={BMC bioinformatics},
		        volume={22},
		        number={1},
		        pages={364},
		        year={2021},
		        publisher={BioMed Central London}
		      }
		      
		      @article{duran2022molecular,
		        title={Molecular dynamics modeling based investigation of the effect of freezing rate on lysozyme stability},
		        author={Duran, Tibo and Minatovicz, Bruna and Bellucci, Ryan and Bai, Jun and Chaudhuri, Bodhisattwa},
		        journal={Pharmaceutical Research},
		        volume={39},
		        number={10},
		        pages={2585--2596},
		        year={2022},
		        publisher={Springer US New York}
		      }
		      
		      @inproceedings{duran2022molecular,
		        title={Molecular Dynamics Modeling Based Investigation of the Effect of Freezing Rate on Protein Stability},
		        author={Duran, Tibo and Minatovicz, Bruna and Bellucci, Ryan and Bai, Jun and Chaudhuri, Bodhisattwa},
		        booktitle={2022 AIChE Annual Meeting},
		        year={2022},
		        organization={AIChE}
		      }
		      
		      @article{bai2022feature,
		        title={Feature fusion Siamese network for breast cancer detection comparing current and prior mammograms},
		        author={Bai, Jun and Jin, Annie and Wang, Tianyu and Yang, Clifford and Nabavi, Sheida},
		        journal={Medical Physics},
		        volume={49},
		        number={6},
		        pages={3654--3669},
		        year={2022},
		        publisher={Wiley Online Library}
		      }
		      
		      @inproceedings{bai2022applying,
		        title={Applying graph convolution neural network in digital breast tomosynthesis for cancer classification},
		        author={Bai, Jun and Jin, Annie and Jin, Andre and Wang, Tianyu and Yang, Clifford and Nabavi, Sheida},
		        booktitle={Proceedings of the 13th ACM International Conference on Bioinformatics, Computational Biology and Health Informatics},
		        pages={1--10},
		        year={2022}
		      }
		      
		      @article{zhou2022combining,
		        title={Combining multi-view ensemble and surrogate lagrangian relaxation for real-time 3D biomedical image segmentation on the edge},
		        author={Zhou, Shanglin and Xu, Xiaowei and Bai, Jun and Bragin, Mikhail},
		        journal={Neurocomputing},
		        volume={512},
		        pages={466--481},
		        year={2022},
		        publisher={Elsevier}
		      }
		      
		      @article{mahdi2022weakly,
		        title={Weakly-Supervised Deep Learning Model for Prostate Cancer Diagnosis and Gleason Grading of Histopathology Images},
		        author={Mahdi Behzadi, Mohammad and Madani, Mohammad and Wang, Hanzhang and Bai, Jun and Bhardwaj, Ankit and Tarakanova, Anna and Yamase, Harold and Hie Nam, Ga and Nabavi, Sheida},
		        journal={arXiv e-prints},
		        pages={arXiv--2212},
		        year={2022}
		      }
		      
		      @inproceedings{hamzehei20233d,
		        title={3D Biological/Biomedical image registration with enhanced feature extraction and outlier detection},
		        author={Hamzehei, Sahand and Bai, Jun and Raimondi, Gianna and Tripp, Rebecca and Ostroff, Linnaea and Nabavi, Sheida},
		        booktitle={Proceedings of the 14th ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics},
		        pages={1--10},
		        year={2023}
		      }
		      
		      @article{bai2024unsupervised,
		        title={Unsupervised feature correlation model to predict breast abnormal variation maps in longitudinal mammograms},
		        author={Bai, Jun and Jin, Annie and Adams, Madison and Yang, Clifford and Nabavi, Sheida},
		        journal={Computerized Medical Imaging and Graphics},
		        volume={113},
		        pages={102341},
		        year={2024},
		        publisher={Elsevier}
		      }
		      
		      @misc{nabavi2023conjoined,
		        title={Conjoined twin network for treatment and analysis},
		        author={Nabavi, Sheida and Yang, Clifford and Bai, Jun},
		        year={2023},
		        month=jul # "~13",
		        publisher={Google Patents},
		        note={US Patent App. 18/096,700}
		      }
		      
		      @inproceedings{bai2022semi,
		        title={Semi-supervised classification of disease prognosis using cr images with clinical data structured graph},
		        author={Bai, Jun and Li, Bingjun and Nabavi, Sheida},
		        booktitle={Proceedings of the 13th ACM international conference on bioinformatics, computational biology and health informatics},
		        pages={1--9},
		        year={2022}
		      }
		      
		      @article{behzadi2024weakly,
		        title={Weakly-supervised deep learning model for prostate cancer diagnosis and Gleason grading of histopathology images},
		        author={Behzadi, Mohammad Mahdi and Madani, Mohammad and Wang, Hanzhang and Bai, Jun and Bhardwaj, Ankit and Tarakanova, Anna and Yamase, Harold and Nam, Ga Hie and Nabavi, Sheida},
		        journal={Biomedical Signal Processing and Control},
		        volume={95},
		        pages={106351},
		        year={2024},
		        publisher={Elsevier}
		      }
		      
		      @inproceedings{agrignan2023deep,
		        title={A deep learning approach for ventricular arrhythmias classification using microcontroller},
		        author={Agrignan, Ya-sine and Zhou, Shanglin and Bai, Jun and Islam, Sahidul and Nabavi, Sheida and Xie, Mimi and Ding, Caiwen},
		        booktitle={2023 24th International Symposium on Quality Electronic Design (ISQED)},
		        pages={1--5},
		        year={2023},
		        organization={IEEE}
		      }
		      
		      @inproceedings{liang2024exploring,
		        title={Exploring latent space for generating peptide analogs using protein language models},
		        author={Liang, Po-Yu and Huang, Xueting and Duran, Tibo and Wiemer, Andrew J and Bai, Jun},
		        booktitle={2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)},
		        pages={842--847},
		        year={2024},
		        organization={IEEE}
		      }
		      
		      @article{jividen2024uncovering,
		        title={Uncovering the Mechanism of Hepatotoxiciy of PFAS Targeting L-FABP Using GCN and Computational Modeling},
		        author={Jividen, Lucas and Duran, Tibo and Niu, Xi-Zhi and Bai, Jun},
		        journal={arXiv preprint arXiv:2409.10370},
		        year={2024}
		      }
		      
		      @article{liang20243,
		        title={E (3)-invariant diffusion model for pocket-aware peptide generation},
		        author={Liang, Po-Yu and Bai, Jun},
		        journal={arXiv preprint arXiv:2410.21335},
		        year={2024}
		      }
		      
		      @misc{nabavi2024conjoined,
		        title={Conjoined twin network for treatment and analysis},
		        author={Nabavi, Sheida and Yang, Clifford and Bai, Jun},
		        year={2024},
		        month=dec # "~26",
		        note={US Patent App. 18/821,269}
		      }
		      
		      @article{hamzehei2025advanced,
		        title={Advanced Feature Extraction and Outlier Detection for 3D Biological/Biomedical Image Registration},
		        author={Hamzehei, Sahand and Bai, Jun and Raimondi, Gianna and Tripp, Rebecca and Ostroff, Linnaea and Nabavi, Sheida},
		        journal={IEEE Transactions on Computational Biology and Bioinformatics},
		        year={2025},
		        publisher={IEEE}
		      }
		      
		      @article{jeny2025hybrid,
		        title={Hybrid transformer-based model for mammogram classification by integrating prior and current images},
		        author={Jeny, Afsana Ahsan and Hamzehei, Sahand and Jin, Annie and Baker, Stephen Andrew and Van Rathe, Tucker and Bai, Jun and Yang, Clifford and Nabavi, Sheida},
		        journal={Medical Physics},
		        year={2025}
		      }
		      
		      @inproceedings{martinez2025robust,
		        title={Robust Training of Deep Learning Models for Mammogram Classification},
		        author={Mart{\'\i}nez-Mart{\'\i}nez, Josu{\'e} and Brown, Olivia and Bai, Jun and Nabavi, Sheida},
		        booktitle={2025 IEEE 22nd International Symposium on Biomedical Imaging (ISBI)},
		        pages={1--4},
		        year={2025},
		        organization={IEEE}
		      }
		</script> 
		
        </div>
	 
	   <div class="col-md-12">    
              <h2 id="committees">Committes</h2>
	    <ul style="list-style: none;">
		    <li>Guest editor &nbsp&nbsp BMC Medical Imaging Special Issue</li>	
		    <li>PC member &nbsp&nbsp 2024 SIAM International Conference on Data Mining (SDM)</li>		
		    <li>PC member &nbsp&nbsp 2022,2023, 2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</li>
 		<li>PC member &nbsp&nbsp 2023 ACM Special Interest Group (SIG) on Knowledge Discovery and Data Mining (KDD)</li>
		 <li>Reviewer &nbsp&nbsp 2023, 2024 MICCAI </li>
		     <li>Reviewer &nbsp&nbsp 2024 NeurIPS</li>
		 <li> Reviewer &nbsp&nbsp 2023 IEEE Engineering in Medicine and Biology Society (EMBC)
		<li>Reviewer &nbsp&nbsp IEEE Transactions on Medical Imaging</li>
		   
		 <li> Reviewer &nbsp&nbsp Applied Artificial Intelligence Journal</li>
		    <li>Reviewer &nbsp&nbsp Neurocomputing</li>
		<li>Reviewer &nbsp&nbsp Medical Physics Journal</li>
		 <li>Reviewer &nbsp&nbsp Current Bioinformatics Journal</li>
		 <li>Co-Reviewer &nbsp&nbsp 2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</li>
	    </ul>
	   </div>
	
       
	
	
	   <div class="col-md-4" >
		<h2 id="projects">Selected Projects</h2>
	     <br><br><a href = "img/firstPaper.png">   
             <img alt="img" src="img/firstPaper.png" class="img-responsive" width="250" height="250"></a>		
           </div>
	   <div class="col-md-5" align="justify"> 
		    <h2></h2>
	    <p> <br><br><strong>Ultraplex Microscopy image reconstruction App development</strong><br />
		The challenge of unraveling the mysterious interplay between individual cells and intricate neural circuits is a daunting task, owing to its overwhelming complexity. However, the current project endeavors to surmount this challenge by creating a state-of-the-art software tool that is capable of seamlessly processing, annotating, registering, analyzing, and reconstructing 3D cells and neural systems with unparalleled precision. This revolutionary tool will serve as the cornerstone for the application of deep learning models and will pave the way for a deeper understanding of the intricacies of the biological world.				
	    </p>				
	   </div>
	   <div class="col-md-4" > 
	     <br><br><br><br><br><br><br><br><a href = "img/secondPaper.png">   
             <img alt="img" src="img/secondPaper.png" class="img-responsive" width="250" height="250"></a>		
           </div>
	   <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Digital Breast Tomosynthesis cancer prediction and localization using Graph Neural Network</strong><br />
		The advent of digital breast tomosynthesis has transformed the landscape of breast imaging diagnosis, thrusting it to unprecedented levels of precision and accuracy. However, despite these remarkable advancements, the automatic detection of breast cancer using the high-resolution, high-volume, and complex images generated by digital breast tomosynthesis has remained a formidable challenge. To conquer this challenge, our research team has embarked on a pioneering journey of developing a revolutionary model for the detection of cancerous 3D mammogram images. In this groundbreaking study, we have developed an innovative model for more precise detection of cancerous 3D mammogram images. This novel model firstly represents 3D mammograms as graphs, then uses a self-attention graph convolutional neural network model to effectively and efficiently learn the features of 3D mammograms, and finally, by utilizing the extracted features, identifies the cancerous 3D mammograms with unparalleled accuracy. This proposed model will undoubtedly pave the way for more efficient and effective breast cancer diagnosis, and a major step forward in the fight against this devastating disease.				
	    </p>				
	   </div>
	 <div class="col-md-4" > 
	     <br><br><br><br><br><br><a href = "img/Third.jpg">   
             <img alt="img" src="img/Third.jpg" class="img-responsive" width="250" height="250"></a>		
         </div>
	 <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Semi-supervised classification of disease prognosis using CR images with clinical data structured graph</strong><br />
		The rapid expansion of global connectivity and urbanization has created a perfect storm for the spread of disease on a worldwide scale. The outbreak of the SARS-COV-2 pandemic has placed immense strain on healthcare systems, particularly in the area of intensive care units. In light of this dire situation, the accurate prognosis of patients' need for intensive care units at the earliest stage of hospital admission has become a top priority, in order to ensure efficient allocation of resources. To this end, we have proposed a groundbreaking clinical data structured graph Markov neural network embedding with computed radiography exam features (CGMNN) to predict the intensive care units demand for COVID-19 patients. This innovative approach utilizes patient chest radiography and clinical data collected at the early stages of hospitalization to make more accurate diagnoses, ensuring that patients receive the care they need in a timely and efficient manner.				
	    </p>				
	 </div>
          <div class="col-md-4" > 
	     <br><br><br><br><br><br><a href = "img/four.jpg">   
             <img alt="img" src="img/four.jpg" class="img-responsive" width="250" height="250"></a>		
          </div>
	  <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Weakly-Supervised Deep Learning Model for Prostate Cancer Diagnosis and Gleason Grading of Histopathology Images</strong><br />
		Prostate cancer is a global scourge, afflicting men worldwide and ranking as the second leading cause of cancer death in the United States. One of the key prognostic features in prostate cancer is the Gleason grading of histopathology images. This grading system, which is based on the tumor architecture of Hematoxylin and Eosin (H&E) stained whole slide images (WSI) is assigned by pathologists and is known to be both time-consuming and prone to inter-observer variability. In recent years, deep learning algorithms have been employed to analyze histopathology images, with promising results in the grading of prostate cancer. However, these algorithms rely on fully annotated datasets, which are both costly and time-consuming to generate. In this work, we propose a novel weakly-supervised algorithm to classify prostate cancer grades. The proposed algorithm is composed of three steps: (1) extracting discriminative areas in a histopathology image by utilizing the Multiple Instance Learning (MIL) algorithm based on Transformers, (2) representing the image by constructing a graph using the discriminative patches, and (3) classifying the image into its Gleason grades by developing a Graph Convolutional Neural Network (GCN) based on the gated attention mechanism. This innovative algorithm is set to revolutionize the field of prostate cancer diagnosis and pave the way for more accurate, efficient, and cost-effective treatment of this devastating disease.<br/>				
	    </p>				
	  </div>
          <div class="col-md-4" > 
	      
             <br><br><video width="250px" height="250px" autoplay loop muted>
         
        	<source src="img/Media1.mp4" type="video/mp4"></video>
    	
          </div>
	  <div class="col-md-5" align="justify"> 
	    <p>  <br><br><strong>Molecular dynamics simulation to uncover the mechanisms of protein instability during freezing</strong><br />
		Freezing is a widely used technique in the pharmaceutical industry for the preservation and transportation of biotherapeutics. In order to gain a deeper understanding of the mechanisms and molecular details of protein destabilization upon freezing, multi-scale molecular dynamics simulations of Lactate dehydrogenase (LDH) protein in phosphate buffer with/without ice formation were performed. The simulations were conducted under both fast and slow ice-growing conditions at 243 K, from one or two sides of the simulation box, respectively. The results showed that the rate of ice formation at all-atom simulations was crucial to the stability of LDH, as faster freezing rates resulted in enhanced structural stability and a higher number of intramolecular hydrogen bonds, less flexible protein residues, lower solvent accessibility, and greater structural compactness. Additionally, protein aggregation was investigated through coarse-grained simulations and was found to be initiated by extended protein structures and retained by electrostatic interactions of salt bridges between charged residues and hydrogen bonds between polar residues of the protein. Lastly, the study of the free energy of dissociation through steered molecular dynamics simulation revealed that LDH was destabilized by the solvation of the hydrophobic core and the loss of hydrophobic interactions. This study, for the first time, provides experimentally validated molecular simulations that reveal the detailed mechanisms of LDH destabilization upon ice formation and cryoconcentration of solutes, and it will help to improve the preservation and transportation of biotherapeutics.				
	    </p>				
	  </div>
	  <div class="col-md-4" > 
	      
             <video width="250px" height="250px" autoplay loop muted>
         
        	<source src="img/Media2.mp4" type="video/mp4"></video>    	
          </div>
          <div class="col-md-5" align="justify">
	  </div>
	  <div class="col-md-4" > 
	      
             <video width="250px" height="250px" autoplay loop muted>
         
        	<source src="img/Media3.mp4" type="video/mp4"></video>   	
          </div>
							     
	 <div class="col-md-12">    
              <h2 id="Demo">Demo</h2>
		 <li><a href="https://colab.research.google.com/drive/13Yp9FI1Pl86wbtGSPZmoeVDl8UToGUeI">Computer vision - Image process teaching demo code</a></li>
		 <li><a download="Mickey-Mouse.jpg" href="img/Mickey-Mouse.jpg" title="ImageName">Click to download test image.</a></li>
		 <li><a href="https://colab.research.google.com/drive/17cwWhE7la4LrmyfHv54lDvW7AUxp5lzx#scrollTo=DBOooCDOiiTR">Python - String teaching demo code</a></li>										 
    		<img alt="ImageName" src="img/Mickey-Mouse.jpg">
</a>
	 </div>
							     
							     
							     
							     
        </div>

    </div>
  
    <!-- Teaching -->
            <div class="col-md-8">    
                <h2 id="teaching">Teaching</h2><br/>
		    <strong>Teaching</strong>
		    <ul>
                <li class="paper" words=""><a href="#">CS 5173/6073 Deep Learning(Fall 2023)</a> <br> This course aims to present the core fundamentals behind the much talked about field of Deep Learning.</li>
                </ul> 		    <strong>Lab Teaching</strong><br/>
		   <ul>
                <li class="paper" words=""><a href="#">CSE 1010 Introduction to Computing for Engineers (Spring 2020, Fall 2021)</a> <br>This course serves as an introduction to the fundamental concepts of computing logic, algorithmic thinking, computing processes, programming languages, and computing environments. </li>
                </ul> 
		 <ul>
                <li class="paper" words=""><a href="#">CSE 1729 Introduction to Principles of Programming (Spring 2021)</a> <br>This course is an introduction to computer programming using a structured programming language. The course will cover the fundamental elements of program design and analysis, and students will learn how to use data and functional abstraction to create correct, efficient, and easy-to-understand programs for a variety of common computing problems. </li>
                </ul> 
		   <strong>Teaching Assistant</strong><br/> 
		 <ul>
			  <li class="paper" words=""><a href="#">CSE 4102 Programming Languages (Fall 2020)</a> <br>This course is focused on the study of programming languages, specifically examining their features and programming paradigms. Topics covered in the course include data types, control structures, run-time environments, and semantics.  </li>
		</ul>
		 <ul>
			  <li class="paper" words=""><a href="#">CSE 3300 Computer Networks and Data Communication (Fall 2019, Spring 2022)</a> <br>This course serves as an introduction to the field of computer networks and data communications. The course will cover the fundamental concepts of network types, components, and topology, protocol architecture, routing algorithms, and network performance. </li>
		</ul>
		 		 <ul>
			  <li class="paper" words=""><a href="#">CSE 2102 Introduction to Software Engineering (Spring 2021)</a> <br>This course covers the fundamental concepts of software engineering including the software life cycle and various software development process models. It will introduce students to the different techniques used in software specification, design methodologies, performance analysis, and verification.</li>
		</ul>
				 <ul>
			  <li class="paper" words=""><a href="#">CPS 542 Database Management Systems,  CPS 562 Advance Database Management Systems (Fall 2018, Spring 2019)</a> <br>This course serves as an introduction to database systems, with a focus on modeling and programming databases. Students will learn about E/R modeling, the theoretical concepts of the relational model, and will gain practical experience working with a commercial DBMS. </li>
		</ul>
            </div>
</div>
    <!-- Other people may like it too! -->
    <a style="color:#b5bec9;font-size:0.8em; float:right;" href="https://github.com/mavroudisv/plain-academic">Plain Academic</a> 
    
</body>

</html>
